api_tests/requests/views/test_request_list_create.py

```diff
@@ -1,189 +0,0 @@
-from unittest import mock
```

Comment:
I split this up simply because it was getting too long and convoluted. It was difficult to isolate the correct test groups.